### PR TITLE
Added 4.1.0 testgrid jobs for scenario tests and u2 support

### PR DESCRIPTION
**Pull request by prasa7**

### CODE REVIEW

File: Integration-Test/master-branch-releases/intg-test-wso2am-testgrid.yaml [hunk 1]
```diff
@@ -8,18 +8,11 @@ infrastructureConfig:
   containerOrchestrationEngine: None
   includes:
     - ADOPT_OPEN_JDK11
-    - ADOPT_OPEN_JDK8
-    - OracleJDK-8
-    - CorrettoJDK-8
-    - CentOS-7.5
     - Postgres-10.5
```
Comment (VimukthiPerera) [inline Integration-Test/master-branch-releases/intg-test-wso2am-testgrid.yaml:11]: Please upgrade to 10.19.

File: Updates-2.0/u2-intg-test-wso2am-4.1.0-full-testgrid.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            WSO2InstanceType: "c3.xlarge"
+scenarioConfigs:
+  - testType: TESTNG
+    remoteRepository: "https://github.com/wso2/testgrid"
+    remoteBranch: master
+    name: integration-tests
+    description: "integration tests"
+    file: jobs/intg-test-resources/update2-releases/run-test.sh
+    inputParameters:
+      WSO2_PRODUCT: "wso2am-4.0.0"
```
Comment (VimukthiPerera) [inline Updates-2.0/u2-intg-test-wso2am-4.1.0-full-testgrid.yaml:45]: Shoudn't this be 4.1.0?

File: WUM/wum-sce-test-wso2am-4.1.0-full-testgrid.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+infrastructureConfig:
+  iacProvider: CLOUDFORMATION
+  infrastructureProvider: AWS
+  containerOrchestrationEngine: None
+  includes:
+    - CentOS-7.5
+    - MySQL-5.7
+    - Oracle-SE2-12.1
+    - SQLServer-SE-14.00
+    - Postgres-10.5
```
Comment (VimukthiPerera) [inline WUM/wum-sce-test-wso2am-4.1.0-full-testgrid.yaml:15]: Please upgrade to 10.19

File: WUM/wum-sce-test-wso2am-4.1.0-full-testgrid.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    - Postgres-12.3
```
Comment (VimukthiPerera) [inline WUM/wum-sce-test-wso2am-4.1.0-full-testgrid.yaml:16]: Please upgrade to 12.9

File: WUM/wum-sce-test-wso2am-4.1.0-full-testgrid.yaml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    - Postgres-13.2
```
Comment (VimukthiPerera) [inline WUM/wum-sce-test-wso2am-4.1.0-full-testgrid.yaml:17]: I am not sure whether this version is also supported by AWS RDS anymore. Please check.

**Review by VimukthiPerera — CHANGES_REQUESTED** [review]

File: WUM/wum-sce-test-wso2am-4.1.0-full-testgrid.yaml (hunk 3 above)
Comment (mushthaq33) [reply WUM/wum-sce-test-wso2am-4.1.0-full-testgrid.yaml:15]: We will need SQLServer-SE-15.00 as well right?

**Review by VimukthiPerera — APPROVED** [review]

**Review by VimukthiPerera — CHANGES_REQUESTED** [review]
Shall we remove postgres 13.2 or upgrade to the newer version?

**Review by mushthaq33 — APPROVED** [review]

**Review by VimukthiPerera — APPROVED** [review]

**Pull request merged**